pep-0546.txt

```diff
@@ -34,37 +35,32 @@ Python Package Index instead is not an acceptable solution.
 PEP 543
 -------
 
-The :pep:`543` defines a new TLS API for Python which would enhance the
-Python security: give access to the root certificate authorities on
-Windows and macOS by using native APIs, instead of OpenSSL. A side effect
-is that it gives access to certificates installed locally by system
-administrators, allowing to use "company certificates" without having to
-modify each Python application and so validate correctly TLS
-certificates (instead of having to ignore or bypass the TLS certificate
-validation).
+:pep:`543` defines a new TLS API for Python which would enhance Python
+security by giving Python applications access to the native TLS implementations
+on Windows and macOS by using native APIs, instead of OpenSSL. A side effect
```

Comment:
you can just drop "by using native APIs", how else would you access a native TLS implementation :-)